docs/cedarling/cedarling-authz.md

```diff
@@ -77,3 +77,101 @@ input = {
 
 decision_result = authz(input)
 ```
+
+## Automatically Adding Entity References to the Context
+
+Cedarling simplifies context creation by automatically including certain entities. This means you don't need to manually pass their references when using them in your policies. The following entities are automatically added to the context:  
+
+- **Workload Entity**  
+- **User Entity**  
+- **Resource Entity**  
+- **Access Token Entity**  
+- **ID Token Entity**  
+- **Userinfo Token Entity**
```

Comment:
Why not just specify the respective snake case here?

Reply:
updated the docs in this commit: [bad2309](https://github.com/JanssenProject/jans/pull/10387/commits/bad230966691ef26f8a343d07d66b61bf5f8762e)